packages/components/document-card/stories/document-card.stories.js

```diff
@@ -10,11 +11,11 @@ storiesOf('ts-document-card', module)
 		const selected = boolean('selected', true);
 		const description = text('description', 'CompanyName');
 		const mobileDescription = text('mobileDescription', 'Description');
-		const cardType = select('cardType', {
-			document: 'document',
-			marketplace: 'marketplace',
-			offer: 'offer',
-			private: 'private'
+		const type = select('type', {
```

Comment:
Can you change it to something like this : https://github.com/Tradeshift/elements/blob/f958046f75d0232f93a568f691658892a7c83c52/packages/components/card/stories/card.stories.js#L20

In this way whenever we are making changes to type it would automatically get updated and we won't miss them in tests

Reply:
done